src/Oceananigans.jl

```diff
@@ -143,7 +143,7 @@ import Base:
 # (libnetcdf) are installed on the machine
 
 "Boolean denoting whether NCDatasets.jl can be loaded on machine."
-const hasnetcdf = @static (Sys.islinux() && Sys.ARCH == :x86_64) ? true : false
+const hasnetcdf = @static (Sys.islinux() && Sys.ARCH == :x86_64) ? false : true

```

Comment:
would

```julia
const hasnetcdf = try
    using NCDatasets
    true
catch
    false
end
```

work?

Reply:
I think we need a comment why linux + x86_64 means we don't have ncdatasets? Does it depend on the status of NetCDF_jll? There might be a relevant issue there to link to

Reply:
Actually, at https://github.com/JuliaBinaryWrappers/NetCDF_jll.jl they claim that they support linux + x86_64. But `libc=glibc`? What does a that mean? Probably it requires a specific C compiler?

Reply:
Shall we add a comment pointing to https://github.com/Alexander-Barth/NCDatasets.jl/issues/150 ?